### PR TITLE
CHANGELOG: Add version 0.6.0

### PR DESCRIPTION
**Pull request by martomi**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,29 @@ to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
...
+- Individual skipped signage points do not trigger notifications anymore. They will be part of daily summary.
+
+### Fixed
+
+- Improved log rotation handling on Windows (thanks [@pieterhelsen](https://github.com/pieterhelsen))
+- Misleading notifications for decreasing plots during copying. The warning threshold is now 2.
+- Wallet notification formatting - received coins are now displayed without scientific notation.
+- More correct handling of skipped signage points - most cases are correctly detected but not all.
+- More accurate daily stats about skipped signage points - now covered by unit test.
+
```
Comment (pieterhelsen) [inline CHANGELOG.md:32]: Missing #171 
Improved Linux install script by installing common dependencies

**Review by pieterhelsen — CHANGES_REQUESTED** [review]
Almost ready to go! One missed fix and potentially missed contribution by ctrlaltdel ?

**Comment by martomi** [comment]
Thanks, should be good to go now!

**Review by pieterhelsen — APPROVED** [review]
🚀

**Pull request merged**